packages/svelte/src/internal/server/dev.js

```diff
@@ -24,7 +24,7 @@ import { HeadPayload, Payload } from './payload.js';
 let parent = null;
 
 /** @type {Set<string>} */
-let seen;
+export let seen;
```

Comment:
this doesn't appear to be used elsewhere

```suggestion
let seen;
```

Reply:
Uh might be an old change.

Reply:
Ohhh it was used in one of the test suites to clear the errors or the second run would not print an expected error reverting